lib/firmata.js

```diff
@@ -1229,10 +1230,19 @@ Board.prototype._sendOneWireRequest = function(pin, subcommand, device, numBytes
 
 Board.prototype.setSamplingInterval = function(interval) {
   var safeint = interval < 10 ? 10 : (interval > 65535 ? 65535 : interval); // constrained
+  this.settings.samplingInterval = safeint;
   this.transport.write(new Buffer([START_SYSEX, SAMPLING_INTERVAL, (safeint & 0xFF), ((safeint >> 8) & 0xFF), END_SYSEX]));
 };
 
 /**
+ * Get sampling interval in millis. Default is 19 ms
+ */
+
+Board.prototype.getSamplingInterval = function(interval) {
+  return this.settings.samplingInterval;
+};
+
```

Comment:
Notes:

- I'd rather not perpetuate the pattern of naming methods `getFoo` and `setFoo`. `setSamplingInterval` was added before I took on maintainership and it's an eye sore. I'm not sure what to do about it now, if anything. I probably have to just accept it. 
- If no `samplingInterval` is specified, this will return `undefined` until some code calls `setSamplingInterval(...)`, but that's not correct, since the default is 19ms. Add `samplingInterval: 19` to `var defaults = {` (line 377)